RetrospectiveExtension.Frontend/index.tsx

```diff
@@ -6,13 +6,15 @@ import { init as sdkInit }  from 'azure-devops-extension-sdk';
 import { isHostedAzureDevOps } from './utilities/azureDevOpsContextHelper';
 import { getProjectId } from './utilities/servicesHelper';
 import './css/main.scss';
+import {  appInsights } from './utilities/external/telemetryClient';
 
 import FeedbackBoardContainer, { FeedbackBoardContainerProps } from './components/feedbackBoardContainer';
-// TODO (enpolat) : import { appInsightsClient, TelemetryEvents } from './utilities/appInsightsClient'
+// import { TelemetryClient } from './utilities/external/telemetryclient';
 
 initializeIcons();
-
-// TODO (enpolat) : appInsightsClient.trackEvent(TelemetryEvents.ExtensionLaunched);
+appInsights.trackEvent({name:"Startup",properties:{"version":"2"}});
+appInsights.trackMetric({name:"startup Time",average:5});
+// let client = TelemetryClient.getClient();
```

Comment:
Mindful of commented out code.

Reply:
Re:tracking versions - It is a great idea to track version information. AppInsights provides an option to set context with addTelemetryInitializers, that way the context of, say a user or version data is tracked consistently throughout. 
Re: tracking startup time - The trackMetric looks like a test code that need to be removed

Reply:
Thanks, it makes sense to me that we should remove the trackMetric code. For tracking version, shouldn't we pull the `version` value from some configuration (for example app insights) object? Also, the `Startup` custom event with `version` property seems confusing to me.